front/src/pages/Home/Home.tsx

```diff
@@ -49,13 +61,14 @@ const Home = () => {
                   <button
                     className={styles.editButton}
                     title="Edit todo"
-                    onClick={() => navigate(`/edit/${todo.id}`)}
+                    onClick={(ev) => handleEditButtonClick(ev, todo.id)}
```

Comment:
proposal for refactoring: to extract this into new component; then you will not need to create inline functions `(ev) => handleEditButtonClick(ev, todo.id)` but instead it would be 

```
<TodoListItem todo={todo} onDelete={handleDeleteButtonClick} onEdit={handleDeleteButtonClick} />
```

Reply:
inline functions are not "bad". But since you passing again and again(it's just 2 times now but imaging additional actions like "bump priority", "lower priority", "clone", "move to other group" etc etc), it suggests "here is code for some specific entity and we have not extracted its code into separate component _yet_". 

Similarly as using the same magic number or string more then once says "and now you _definitely_ want to have this as a named constant".

I also think `ev.stopPropagation` should happen inside the `TodoListItem`. And Home will just make either navigation to "Edit" sreen or call deletion logic.